### PR TITLE
[BWA-27] Sync translations with Crowdin

### PR DESCRIPTION
**Pull request by KatherineInCode**
## 🎟️ Tracking

[BWA-27](https://bitwarden.atlassian.net/browse/BWA-27)

## 📔 Objective

This starts syncing translations with [Crowdin](https://crowdin.com/project/bitwarden-authenticator-mobile) for managing translations.

The workflow changes match the current mobile repo's workflows as near as I can tell.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[BWA-27]: https://bitwarden.atlassian.net/browse/BWA-27?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by fedemkr — DISMISSED** [review]
LGTM

File: crowdin.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+project_id_env: _CROWDIN_PROJECT_ID
+api_token_env: CROWDIN_API_TOKEN
```
Comment (withinfocus) [inline crowdin.yml:2]: ⛏️ Two different variable formats (leading underscore).

File: .github/workflows/crowdin-pull.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    inputs: {}
+  schedule:
+    - cron: '0 0 * * 5'
+
+jobs:
+  crowdin-sync:
+    name: Autosync
+    runs-on: ubuntu-20.04
+    env:
+      _CROWDIN_PROJECT_ID: "673718"
```
Comment (withinfocus) [inline .github/workflows/crowdin-pull.yml:15]: ❓ Is it worth these being secrets, or constants somewhere central?

File: .github/workflows/crowdin-pull.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    steps:
+      - name: Checkout repo
```
Comment (withinfocus) [inline .github/workflows/crowdin-pull.yml:17]:
```suggestion
      - name: Check out repo
```

File: .github/workflows/crowdin-pull.yml (hunk 2 above)
Comment (KatherineInCode) [reply .github/workflows/crowdin-pull.yml:15]: The project ID is public, and the BWA and PM have different project IDs, so I think it's fine as it is now.

File: crowdin.yml (hunk 1 above)
Comment (joseph-flinn) [reply crowdin.yml:2]: `_CROWDIN_PROJECT_ID` is a job level environment variable versus a step level. On smaller workflows, this pattern is marginal, but on workflows with 500-1000 lines, the differentiation helps with maintenance.

The workflow linter rule can be found here: https://github.com/bitwarden/workflow-linter/blob/f204a10c9bb6e623f8ab2223a9b577bcf94f46c1/src/bitwarden_workflow_linter/rules/job_environment_prefix.py#L19

**Review by joseph-flinn — APPROVED** [review]

**Review by SaintPatrck — APPROVED** [review]

**Comment by KatherineInCode** [comment]
Checkmarx finds are for internal actions, which we control. Marking as "Proposed Not Exploitable" in Checkmarx.

**Pull request merged**